app/views/lists/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Lists</h1>
+
+
+<% @lists.each do |list| %>
+<div>
+  <h2><%= list.title %></h2>
+  <p> <%= list.description %> </p>
+  <% list.books.each_with_index do |book, index| %>
```

Comment:
@samnang this is exactly what we were looking for. We were contemplating if we should use SQL directly or active record associations.